products/sle/main.pm

```diff
@@ -701,6 +701,7 @@ sub load_baremetal_tests {
     }
     elsif (get_var('NFV')) {
         mellanox_config();
+        loadtest "kernel/mellanox_ofed" if check_var('OFED', '1');
```

Comment:
if without OFED_URL test will not work anyway , maybe you can simplify things and just check here for OFED_URL so you would have one variable less ?

Reply:
good point, will do so. Thanks.